### PR TITLE
feat(card): Implement Admiral Motti and Snowspeeder

### PR DESCRIPTION
**Pull request by joshfullmer**

### CODE REVIEW

File: server/game/cards/01_SOR/units/AdmiralMottiBrazenAndScornful.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    protected override getImplementationId() {
+        return {
+            id: '9996676854',
+            internalName: 'admiral-motti#brazen-and-scornful'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenDefeatedAbility({
+            title: 'Ready a villainy unit',
```
Comment (AMMayberry1) [inline server/game/cards/01_SOR/units/AdmiralMottiBrazenAndScornful.ts:15]: nit: our practice is to capitalize the aspect names in ability titles

File: server/game/cards/01_SOR/units/Snowspeeder.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addOnAttackAbility({
+            title: 'Exhaust an enemy Vehicle ground unit',
+            targetResolver: {
+                locationFilter: Location.GroundArena,
+                controller: RelativePlayer.Opponent,
+                cardCondition: (card) => card.isUnit() && card.traits.has(Trait.Vehicle),
```
Comment (AMMayberry1) [inline server/game/cards/01_SOR/units/Snowspeeder.ts:19]: No functional issue here but do you mind using `card.hasSomeTrait(Trait.Vehicle)`. That method is more flexible (can accept an array) so I want to use that one consistently so people know it's there

File: test/server/cards/01_SOR/units/AdmiralMottiBrazenAndScornful.spec.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                    },
+                    player2: {
+                        groundArena: [{ card: 'cell-block-guard', exhausted: true }]
+                    }
+                });
+            });
+
+            it('should allow player to ready an imperial unit', function() {
+                // Attacks into cell block guard as the only valid attack
+                this.player1.clickCard(this.admiralMotti);
```
Comment (AMMayberry1) [inline test/server/cards/01_SOR/units/AdmiralMottiBrazenAndScornful.spec.js:19]: Can we add `expect(this.player1).toHavePassAbilityButton();`

File: test/server/cards/01_SOR/units/Snowspeeder.spec.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            });
+
+            it('should exhaust an enemy Vehicle ground unit', function() {
+                this.player1.clickCard(this.snowspeeder);
+                this.player1.clickPrompt('Ambush');
+
+                expect(this.atst.exhausted).toBe(true);
+            });
+
+            describe('when multiple valid targets,', function() {
```
Comment (AMMayberry1) [inline test/server/cards/01_SOR/units/Snowspeeder.spec.js:26]: We try to not overdo it on tests to keep our github actions usage low (the tests are extremely heavy for some reason, not sure if it's jasmine or the test setup). We typically don't bother testing both the single-target and multi-target case, do you mind removing one.

If you choose to keep the single-target case, please add a comment explaining that it is automatically targeting due to sentinel

**Comment by joshfullmer** [comment]
@AMMayberry1 thanks for the feedback! Should all be addressed in https://github.com/SWU-Karabast/forceteki/pull/93/commits/b0e9f745e0685c383ae09717af325fec8600ffb6

**Review by AMMayberry1 — APPROVED** [review]

**Pull request merged**